dask_cuda/benchmarks/local_cupy.py

```diff
@@ -28,75 +29,121 @@ async def _run(client, args):
     rs = da.random.RandomState(RandomState=xp.random.RandomState)
 
     if args.operation == "transpose_sum":
+        rng = start_range(message="make array(s)", color="green")
         x = rs.random((args.size, args.size), chunks=args.chunk_size).persist()
         await wait(x)
+        end_range(rng)
```

Comment:
Given it's only two lines, I'm not sure it's worth the trouble, but feels like the `start_range`/`end_range` would be a perfect candidate for a decorate function. Just saying for future consideration, no action required. 🙂

Reply:
Or using [`contextmanager`]( https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager )?

cc @shwina (in case you have thoughts on how to do this 🙂)

Reply:
I was considering the same - I'll look into that if we end up extending the benchmark again 😁